src/openrct2.c

```diff
@@ -75,7 +75,7 @@ EVP_MD_CTX *gHashCTX = NULL;
 int _finished;
 
 // Used for object movement tweening
-static struct { sint16 x, y, z; } _spritelocations1[MAX_SPRITES], _spritelocations2[MAX_SPRITES];
+static sprite_coords _spritelocations1[MAX_SPRITES], _spritelocations2[MAX_SPRITES];
```

Comment:
Just use rct_xyz16... or are you going to tell me that packing will make it slower?